sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizeCustomEntitiesAction.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Configurations that allow callers to specify details about how to execute
+    /// a Recognize Custom Entities action in a set of documents.
+    /// Project Name and Deployment Name are required.
+    /// </summary>
+    public class RecognizeCustomEntitiesAction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RecognizeCustomEntitiesAction"/>
+        /// class which allows callers to specify details about how to execute
+        /// a Recognize Custom Entities action in a set of documents.
+        /// Sets the <see cref="ProjectName"/> and <see cref="DeploymentName"/> properties
+        /// </summary>
+        public RecognizeCustomEntitiesAction(string projectName, string deploymentName)
```

Comment:
In the rest of the client we usually add `param` tags in the docstrings for each parameter in public constructors/methods. Such as in:
```cs
        /// <summary>
        /// Initializes a new instance of the <see cref="DocumentAnalysisClient"/> class.
        /// </summary>
        /// <param name="endpoint">The endpoint to use for connecting to the Form Recognizer Azure Cognitive Service.</param>
```
In this case I would just copy the descriptions of the `ProjectName` and `DeploymentName` properties.